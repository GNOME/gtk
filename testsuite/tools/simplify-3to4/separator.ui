<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkVSeparator">
    <property name="visible">True</property>
  </object>
  <object class="GtkHSeparator">
    <property name="visible">True</property>
  </object>
</interface>
