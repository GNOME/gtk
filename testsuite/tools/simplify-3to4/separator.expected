<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkSeparator">
    <property name="orientation">vertical</property>
  </object>
  <object class="GtkSeparator"/>
</interface>
